src/models/Popup.js

```diff
@@ -52,6 +52,7 @@ export default class Popup extends Base {
       }
       return ''
     })()
+    this.hasTransition = !!this.transitionEnd;
```

Comment:
I prefer explicit cast as:
```suggestion
    this.hasTransition = Boolean(this.transitionEnd);
```